fdbserver/TLogGroup.actor.cpp

```diff
@@ -221,6 +228,15 @@ void TLogGroupCollection::storeState(CommitTransactionRequest* recoveryCommitReq
 		       BinaryWriter::toValue(group->id(), Unversioned()));
 		TraceEvent("TLogGroupSSTeamAssignSave").detail("GroupID", group->id()).detail("SSTeamId", teamId);
 	}
+
+	for (const auto& [splitKey, storageTeamID] : splitKeyStorageTeamIDMapping) {
```

Comment:
This introduces a new map in txnStateStore, which could be avoided. Instead, there are already a mapping of `keyServers`: keyRange -> set of SSes, and a mapping of `serverKeys`: SS -> keyRanges. So key ranges defined by split keys can just reuse these two existing mappings.